pkg/api/serialization_test.go

```diff
@@ -458,14 +480,14 @@ func TestSpecificKind(t *testing.T) {
 	}
 	seed := int64(2703387474910584091) //rand.Int63()
 	for i := 0; i < fuzzIters; i++ {
-		t.Logf(`About to test %v with "v1"`, kind)
+		//t.Logf(`About to test %v with "v1"`, kind)
 		fuzzInternalObject(t, v1.SchemeGroupVersion, item, seed)
 		roundTrip(t, kapi.Codecs.LegacyCodec(v1.SchemeGroupVersion), item)
 	}
 }
 
 // Keep this in sync with the respective upstream set
-var nonInternalRoundTrippableTypes = sets.NewString("List", "ListOptions")
+var nonInternalRoundTrippableTypes = sets.NewString("List", "ListOptions", "WatchEvent")
```

Comment:
Can you link to a description of why this is not roundtrippable? We already have an issue for eliminating this list: https://github.com/openshift/origin/issues/5529